frontend/public/components/_filter-toolbar.scss

```diff
@@ -23,6 +23,6 @@
 }
 
 .co-filter-search--full-width {
-    flex-grow: 1;
-    max-width: 90vw;
+  flex-grow: 1;
+  max-width: 370px;
```

Comment:
I think we at least need to change the name of this class if it no longer fills the width. Do we need to restrict it so much? It's a shame we're losing space to type filters.

Reply:
@spadgett we don't need to restrict it. Without any css changes, the container of the filter was 90vw which was putting the table column icon to the far right even though the filter wasn't filling up all that horizontal space. (see the pic) 

This simply keeps the container the same size as the component, we are not losing any filter text.

![Screen Shot 2020-07-10 at 3 45 57 PM](https://user-images.githubusercontent.com/35978579/87192448-77f2c980-c2c4-11ea-93d9-987d43fc0203.png)